lightkurve/tests/test_lightcurve.py

```diff
@@ -173,6 +173,69 @@ def test_lightcurve_append_multiple():
     assert_array_equal(lc.time, 4*[1, 2, 3])
 
 
+def test_lightcurve_copy():
+    """Test ``LightCurve.copy()``."""
+    time = np.array([1, 2, 3, 4])
+    flux = np.array([1, 2, 3, 4])
+    error = np.array([0.1, 0.2, 0.3, 0.4])
+    lc = LightCurve(time=time, flux=flux, flux_err=error)
+
+    nlc = lc.copy()
+    assert_array_equal(lc.time, nlc.time)
+    assert_array_equal(lc.flux, nlc.flux)
+    assert_array_equal(lc.flux_err, nlc.flux_err)
+
+    np.put(nlc.time, 1, 5)
```

Comment:
Just curious: is there a reason behind using `np.put(nlc.time, 1, 5)` instead of the slightly friendlier `nlc.time[1] = 5` syntax?

Reply:
Not really, the `np.put(a, x, y)` method is specific to `numpy`, just like `assert_array_equal(a, b)`. 

I also tested `nlc.time[x] = y` and it seemed to work fine, but switched it in case `numpy` arrays are different from lists in some way.

`np.nan` was an interesting example: a floating-point object that means nothing yet cannot match itself.